simexpal/base.py

```diff
@@ -134,7 +134,11 @@ def construct_instances():
 
 		def construct_variants():
 			if 'variants' in self.yml:
+				check_key_value(self.yml['variants'], 'axis')
```

Comment:
Since we know the key here, can't we simply pass `self.yml['variants']['axis']`? (Same for the other cases below.)

Reply:
done (see remark above)